src/System.Numerics.Vectors/tests/GenericVectorTests.netcoreapp.cs

```diff
@@ -212,15 +284,129 @@ public partial class GenericVectorTests
         [Fact]
         public void ConstructorWithUnsupportedTypes_DateTime() => TestConstructorWithUnsupportedTypes<DateTime>();
         [Fact]
-        public void ConstructorWithUnsupportedTypes_Char() => TestConstructorWithUnsupportedTypes<char>();
+        public void ConstructorWithUnsupportedTypes_Char() => TestConstructorWithUnsupportedTypes<Char>();
```

Comment:
Why the change to uppercase `char`? (also, `bool` is another good one to test)

Reply:
The TT somehow started emitting the actual type names instead of the friendly names so I had to make some tweaks to the TT itself. I thought I had fixed all occurrences but apparently I missed one. :)